runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/internal/registry/ConfigureTask.java

```diff
@@ -300,4 +296,35 @@ else if ("http".equals(configURL.getProtocol()) || "https".equals(configURL.getP
 
         return null;
     }
+
+
+    private String readURL(URL url) throws IOException, URISyntaxException, InterruptedException
```

Comment:
Please remove `throws` clause and use `rethrowUnchecked` in this method body instead.